server/resp/src/test/java/org/infinispan/server/resp/RespTwoNodeTest.java

```diff
@@ -173,4 +177,17 @@ public void unsubscribed(String channel, long count) {
 
       return handOffQueue;
    }
+
+   @Test
+   public void testBlpop() throws InterruptedException, ExecutionException {
```

Comment:
Should we include some more here? Installing a listener from client1 but pushing values from client2.

Reply:
ok, added some tests with mixed cases